docker-compose-ode.yml

```diff
@@ -57,18 +57,20 @@ services:
       - "5555:5555/udp"

```

Comment:
Update all tag references from `2025-q1` to `2025-q2`. Consider pulling this config out of the docker-compose and instead reference an env variable. Also, is it required to have the conflictmonitor as a service? If so, we may still need to hold off on this release.